traefik/faency
# feat: button for interactive badge

## Description
- Fixes https://github.com/traefik/faency/issues/304

Use `button` for interactive badge.

## How to test?

Check the story, make sure that badge with `interactive` props are buttons and the other still span.

## Good PR checkboxes


## Good Review checkboxes

<details>
<summary> ℹ️  Copy the snippet and paste in the review field to fill it</summary>

```markdown
```

</details>
